src/styles.ts

```diff
@@ -1,6 +1,6 @@
 const iFrameInitialStyle = {
   border: 'none',
-  position: 'absolute',
+  position: 'fixed',
```

Comment:
set the zIndex also here to some higher number.
@karthik-durai

Reply:
[done](https://github.com/arcana-network/auth/pull/174/commits/f343ec02e439bc993801c3ac35efee7c2c1fc51f).